source/colors/types.d.ts

```diff
@@ -1,8 +1,8 @@
-type TColorValuePart = string | number;
+type ColorValuePart = string | number
```

Comment:
Keep using types.d.ts? Or move things around to ts files and import types when needed?

Reply:
Second option please